### PR TITLE
Some updates to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by bantonsson**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,9 +1,15 @@
-## Contributing
+# Contributing
 
 Pull requests for bug fixes are welcome, but before submitting new features or changes to current functionality [open an issue](https://github.com/DataDog/dd-trace-java/issues/new)
 and discuss your ideas or propose the changes you wish to make. After a resolution is reached a PR can be submitted for review.
 
-### Code Style
+## Requirements
+
+To build the full project from the commande line you need to have JDK versions for 7,8,11, and 14 installed on your machine, as well as the following environment variables set up: `JAVA_7_HOME, JAVA_8_HOME, JAVA_11_HOME, JAVA_14_HOME`, pointing to the respective JDK.
```
Comment (richardstartin) [inline CONTRIBUTING.md:8]: typo: command line

Comment (bantonsson) [reply CONTRIBUTING.md:8]: Thanks for catching my French 😉

Comment (richardstartin) [reply CONTRIBUTING.md:8]: pas de soucis :)

**Comment by bantonsson** [comment]
So is the `test_zulu14` a know issue?

**Pull request merged**